katalog/nginx/bases/controller/DaemonSet-ingress-nginx-controller.yml

```
@@ -98,16 +106,37 @@ spec:
             periodSeconds: 10
             successThreshold: 1
             timeoutSeconds: 1
+          ports:
+            - name: http
+              containerPort: 8080
+              protocol: TCP
+            - name: https
+              containerPort: 8443
```

If you drop the flags above setting the custom ports, you need to change the ports like this:

```suggestion
              containerPort: 80
              protocol: TCP
            - name: https
              containerPort: 443
```

---

Thanks, but I'll leave the ports as they were before 😄